app/views/posts/index.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Here is a list of all posts for a given user</h1>
+<section id="users">
+  <div class="user">
+    <div class="avatar">
+      avatar
+    </div>
+    <div class="info">
+      <h2><%= @user.name %></h2>
+      <span>Number of posts: <%= @user.posts_counter %></span>
+    </div>
+  </div>
+</section>
+<section id="recent-posts">
+  <%@posts.each_with_index do |post, index|%>
+    <div class="post">
+      <h4><%= link_to "Post ##{index+1}", user_post_path(@user, post)%></h4>
+      <p>
+        <%= link_to post.title, user_post_path(@user, post) %>
+      </p>
+    </div>
+    <% if post.comments.length.positive? %>
+      <div class="comments">
+        <% post.comments.each do |comment| %>
```

Comment:
- [ ] We were asked to render the five most recent comments(post.recent_five.each) for each post. Right now you are rendering all comments for each post. Can we rectify the minute issue?


Only the show page should display all the comments